@jchauncey Looks like there is a static validation test failure: 'pkg/tiller/hooks.go:114:3:warning: ineffectual assignment to hw (ineffassign)'. Could you address that and repush?

--- a/docs/charts_hooks.md
+++ b/docs/charts_hooks.md
@@ -116,7 +116,7 @@ metadata:
     # This is what defines this resource as a hook. Without this line, the
     # job is considered part of the release.
     "helm.sh/hook": post-install
-    "helm.sh/hookWeight": "-5"
+    "helm.sh/hook-weight": "-5"
 spec:
   template:
     metadata:
@@ -161,7 +161,7 @@ It is also possible to define a weight for a hook which will help build a determ
 
 ```
   annotations:
-    "helm.sh/hookWeight": "5"
+    "helm.sh/hook-weight": "5"
 ```
 
 Hook weights can be positive or negative numbers but must be represented as strings. When Tiller starts the execution cycle of hooks of a particular Kind it will sort those hooks in ascending order. 

--- a/pkg/hooks/hooks.go
+++ b/pkg/hooks/hooks.go
@@ -24,7 +24,7 @@ import (
 const HookAnno = "helm.sh/hook"
 
 // HookWeightAnno is the label name for a hook weight
-const HookWeightAnno = "helm.sh/hookWeight"
+const HookWeightAnno = "helm.sh/hook-weight"
 
 // Types of hooks
 const (

--- a/pkg/tiller/hooks.go
+++ b/pkg/tiller/hooks.go
@@ -111,9 +111,8 @@ func sortManifests(files map[string]string, apis chartutil.VersionSet, sort Sort
 			continue
 		}
 
-		hw := 0
 		hws, _ := sh.Metadata.Annotations[hooks.HookWeightAnno]
-		hw, err = strconv.Atoi(hws)
+		hw, err := strconv.Atoi(hws)
 		if err != nil {
 			hw = 0
 		}

